integration-test/concat/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM golang:1.7.1
+
+# install gx and gx-go
+RUN go get github.com/whyrusleeping/gx
+RUN go get github.com/whyrusleeping/gx-go
+
+# Clone concat repo
+RUN mkdir -p /go/src/github.com/mediachain
+RUN git clone https://github.com/mediachain/concat /go/src/github.com/mediachain/concat
+WORKDIR /go/src/github.com/mediachain/concat
+
+ARG concat_ref=master
+RUN git checkout ${concat_ref}
+
+# install gx dependencies
```

Comment:
Why duplicate these instead of running concat/setup.sh directly?
It's already getting out of date, the datastore branch has a nasty rocksdb dep for instance.